CodeAssignment_MVVM_Combine/Sources/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,321 @@
+//
+//  DetailViewController.swift
+//  CodeAssignment_MVVM_Combine
+//
+//  Created by gyusoku.i on 2024/05/16.
+//
+
+import UIKit
+import SnapKit
+import SDWebImage
+
+private enum Const {
+    /// layout設定で使うleftPadding
+    static let leftPadding: CGFloat = 20
+    /// layout設定で使うRightPadding
+    static let rightPadding: CGFloat = 20
+    /// starsに関するString型の説明文(スペース入り)
+    static let starsExplainString: String = " stars"
+    /// watchersに関するString型の説明文
+    static let watchersExplainString: String = " watchers"
+    /// forksに関するString型の説明文
+    static let forksExplainString: String = " forks"
+    /// openIssuesに関するString型の説明文
+    static let openIssuesExplainString: String = " issues"
+    /// DetailViewControllerで表すLanguageColorViewのheight サイズ
+    static let colorViewHeightSize: CGFloat = 20
```

Comment:
```suggestion
    static let colorViewHeight: CGFloat = 20
```
Sizeって入れなくてもHeightが「高さ」なので伝わるね

Reply:
ここも[b6ed2d8]にて修正しました！ご確認お願いします！